### DISCUSSION TITLE
Return Youtube Likes on Safari

### DISCUSSION THREAD

**Discussion by gradygambrell** [Anarios/return-youtube-dislike#715] [Q&A]
How do you enable Return Youtube Likes on Safari 15.6 on Monterey 12.5. Is it supported it yet?

**Comment by infinitepower18** [comment]
You can build it for Safari yourself, as long as you have Xcode installed

1. Download the source code of the Chrome extension using https://crxextractor.com/ and extract the zip
2. Run the following command in terminal `xcrun safari-web-extension-converter --macos-only /path/to/extension --copy-resources`
3. In Xcode click Product located on the menu bar and select Archive
4. Click Distribute App, Development, select your team, automatically manage signing, export to Applications folder
5. Open the app from launchpad, click the button to open Safari Preferences, enable the extension

It should work after you done all these steps.

<img width="1800" alt="Screenshot 2022-08-24 at 7 53 21 pm" src="https://user-images.githubusercontent.com/44692189/186436511-7cf147f3-a242-490b-b150-d2de90c776d2.png">

**Comment by Julmik647** [comment]
https://apple.co/3B7zfFr

**Comment by Julmik647** [comment]
Try https://apple.co/3B7zfFr but it's not offical
